fiat-roles/src/main/java/com/netflix/spinnaker/fiat/config/FiatAdminConfig.java

```diff
@@ -0,0 +1,21 @@
+package com.netflix.spinnaker.fiat.config;
+
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+@Configuration
+@ConfigurationProperties("fiat")
+public class FiatAdminConfig {
+
+    private AdminRoles admin = new AdminRoles();
+
+    @Data
+    public static class AdminRoles {
+        private List<String> adminRoles = new ArrayList<>();
```

Comment:
The context of the `adminRoles` field is in the `AdminRoles` class, so you could just name it `roles`. That way, your config would look something like:

```
fiat:
  admin:
    roles:
      - foo
      - bar
  baz: qux (etc)
```